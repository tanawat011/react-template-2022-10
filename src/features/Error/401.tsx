export const Error401 = () => {
  return <div>401 Unauthorized</div>
}
