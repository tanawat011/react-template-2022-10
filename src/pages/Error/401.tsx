import { allRoutes } from 'Routes'
import { Button } from 'components/Button'

export const Error401 = () => {
  return (
    <div>
      <p>401 Unauthorized</p>

      <Button link to={allRoutes.auth.fullPath}>
        Back to Login
      </Button>
    </div>
  )
}
