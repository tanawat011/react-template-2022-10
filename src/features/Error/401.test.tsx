import { render } from '@testing-library/react'

import { Error401 } from './401'

describe('<Error401 />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<Error401 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
