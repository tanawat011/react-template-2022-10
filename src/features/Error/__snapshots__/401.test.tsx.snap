// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error401 /> match snapshot 1`] = `
<DocumentFragment>
  <div>
    401 Unauthorized
  </div>
</DocumentFragment>
`;
